readme.txt

```diff
@@ -28,8 +28,8 @@ With the official AMP plugin for WordPress, the WordPress ecosystem is provided
 == Installation ==
 
 1. Upload the folder to the `/wp-content/plugins/` directory.
-1. Activate the plugin through the 'Plugins' menu in WordPress.
-1. If you currently use older versions of the plugin in `Classic mode`, it is strongly encouraged to migrate to `Paired` or `Native mode`.
+2. Activate the plugin through the 'Plugins' menu in WordPress.
+3. If you currently use older versions of the plugin in `Reader` mode, it is strongly encouraged to migrate to `Transitional` or `Native` mode.
```

Comment:
The note about the behavior of reader/transitional modes should also be added here, that they are for mobile visitors from Google Search, Twitter, etc. No redirection is done. This is not a mobile theme.

/cc @amedina

Reply:
Agree. Same line from the Admin Settings page would work.